plugins/flutter_aepedgeidentity/lib/src/aepedgeidentity_identity_map.dart

```diff
@@ -18,7 +18,7 @@ class IdentityMap {
   Map<String, List<IdentityItem>> identityMap = {};
 
   void addItem(IdentityItem item, String namespace) {
-    if (item.id.isEmpty || item.id.length == 0) {
```

Comment:
No null check here?

Reply:
item is not nullable, we don't need to check null here.